network/src/commonJvmAndroid/kotlin/com/wire/kalium/network/HttpEngine.kt

```diff
@@ -103,5 +103,9 @@ private fun OkHttpClient.Builder.ignoreAllSSLErrors() {
 
 fun supportedConnectionSpecs(): List<ConnectionSpec> {
     val wireSpec = ConnectionSpec.Builder(ConnectionSpec.RESTRICTED_TLS).build()
-    return listOf(wireSpec, ConnectionSpec.CLEARTEXT)
```

Comment:
The `ConnectionSpec.CLEARTEXT` was already added, so not sure why the new Http client is needed?

Reply:
because wireSpec has `RESTRICTED_TLS`

Reply:
Which will only allow connections through HTTPS